### DISCUSSION TITLE
[M-Z] (10 Febbraio 2020) Esercizio 8

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#474] [Esami & Esoneri M-Z]
### Traccia

<img width="900" alt="image" src="https://github.com/user-attachments/assets/85fa8ade-de54-444e-8a23-890ec19cf92a">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20200210.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20200210.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/ce79ab01-dda5-490a-84ca-965583edb2e8)
